dotnet-authserver/tests/TeacherIdentity.AuthServer.Tests/HostFixture.cs

```diff
@@ -85,6 +87,15 @@ public void ResetMocks()
     // N.B. Don't call this from InitializeAsync - it won't work
     public void SetUserId(Guid? userId) => Services.GetRequiredService<CurrentUserIdContainer>().CurrentUserId.Value = userId;
 
+    public void ResetUseNewTrnLookupJourney()
```

Comment:
Hoped to just reset everything in FindALostTrnIntegration config section, but that was too hard/messy